contrib/net/http/roundtripper_test.go

```diff
@@ -88,6 +89,12 @@ func TestRoundTripper(t *testing.T) {
 	assert.Equal(t, true, s1.Tag("CalledAfter"))
 	assert.Equal(t, ext.SpanKindClient, s1.Tag(ext.SpanKind))
 	assert.Equal(t, "net/http", s1.Tag(ext.Component))
+	assert.Equal(t, "127.0.0.1", s1.Tag(ext.NetworkDestinationName))
+
+	wantPort, err := strconv.Atoi(strings.TrimPrefix(s.URL, "http://127.0.0.1:"))
+	require.NoError(t, err)
+	require.NotEmpty(t, wantPort)
+	assert.Equal(t, wantPort, s1.Tag(ext.NetworkDestinationPort))
```

Comment:
Curious why we're mixing require and assert in the same test?

Reply:
I'm using `require` when I don't want the test to continue in case the assertion fails. Like for example in case you are gonna make a bunch of assertions on a struct that could be `nil`, so you avoid seeing a lot of panics in your test, which might be misleading.

In this case I don't want to have the chance of the assertion of line 97 to pass with wantPort = 0 in case the URL from the `httptest` server is something I don't expect.